code/_onclick/item_attack.dm

```diff
@@ -53,6 +53,8 @@
 	//	M.lastattacker = null
 	/////////////////////////
 
+	if(istype(M, /mob/living/simple_animal))
```

Comment:
I'm now starting to wonder if there should even be any `/mob/` let alone `/mob/living/` that allows spam clicking.